addons/event/static/src/js/tours/event_tour.js

```diff
@@ -24,6 +24,8 @@ var _t = core._t;
 var tour = require('web_tour.tour');
 var EventAdditionalTourSteps = require('event.event_steps');
 
+const inEventForm = '*[name="event_type_id"]';
```

Comment:
Maybe add a small comment explaining why we use this instead of a proper class (stable blabla).
And can we replace '*' by 'input' or something?

Reply:
@awa-odoo Good idea :) 
No, because in readonly mode it will be a `<a/>` :/